site/gatsby-site/README.md

```diff
@@ -149,7 +149,8 @@ npm run codegen
 
 ### Schema and API Stitching
 
-The API previously relied on the MongoDB Realm GraphQL API to fetch data, but now that it has been deprecated, we have migrated to a new API. During the migration from Realm to our API, we stitch two GraphQL schemas: one from Atlas and one local. These schemas can be found in the `remote.ts` and `local.ts` files respectively. The final combined schema is found in `schema.ts`.
+[!IMPORTANT]
```

Comment:
```suggestion
> [!IMPORTANT]
```

oops. See if this renders now?
https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts